### ISSUE TITLE
WCAG 1.4.3 Contrast

### ISSUE THREAD

**Issue by kosarko** [ufal/dspace-angular#14]
check the contrast of background vs foreground in lighthouse (are there issues with other components than footer)?

**Comment by ToneMereteBruvik** [comment]
There are according to Lighthouse issues not only in the footer (testet today), some of these are failing on only regular text, but are ok with large text. Good contrast is very important for everyone.

The list of problems reported by Lighthouse starts like this:

![Image](https://github.com/user-attachments/assets/a4e9c1a1-296c-459f-af5d-193c7c7a0236)
